pkg/apb/types.go

```diff
@@ -88,9 +88,9 @@ const (
 	ApbRole = "cluster-admin"
 
 	// Bind credential gathering constants
-	// 5 seconds x 60 retries = 5 minute timout
+	// 5 seconds x 9000 retries = 30 minute timeout
 	credentialExtInterval = 5
-	credentialExtRetries  = 60
+	credentialExtRetries  = 9000
```

Comment:
Could we bump this to 2 hours.
I think we might go over 30 mins with upcoming EMR work.